eth-contracts/contracts/service/DelegateManager.sol

```diff
@@ -141,6 +150,10 @@ contract DelegateManager is RegistryContract {
         if (!delegatorExistsForSP(delegator, _targetSP)) {
```

Comment:
not sure if this *actually* matters, but shouldn't all event emission be at the end of function, or where they logically make sense

Reply:
agreed